### PR TITLE
Blog post: Backstage turns two

### PR DESCRIPTION
**Pull request by vinzscam**

### CODE REVIEW

**Review by fengypants — COMMENTED** [review]
Looking good to me! Though, a festive GIF should be coming your way to replace the PNG. And since the big day is tomorrow, I believe we are holding off on merging until the morning.

**Review by Rugvip — APPROVED** [review]
🍰 🎉 :shipit:

**Review by freben — APPROVED** [review]
Yay! 🎉🎉

**Review by jhaals — APPROVED** [review]
🎉🎉

**Review by benjdlambert — APPROVED** [review]
:tada:

File: microsite/blog/2022-03-16-backstage-turns-two.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+title: Backstage Turns Two!
+author: Lee Mills, Spotify
+authorURL: https://github.com/leemills83
+authorImageURL: https://avatars1.githubusercontent.com/u/1236238?s=460&v=4
+---
+
+![A cake celebrating Backstage’s second birthday](assets/22-03-16/backstage-2nd-birthday.gif)
+
+_TLDR;_ Backstage has turned two. Of course, this is software, so we expect things to move fast. But when you have a passionate, worldwide community building together, we’ve now seen firsthand just how fast that can be. We’re marking our second year as a community by becoming a CNCF incubating project, reaching 100 public adopters, releasing version 1.0 of the core app, and other very nice birthday gifts. Rather than a toddler finding their feet, Backstage today is beginning to look awfully grown-up for a two-year-old.
```
[the remark this answers is not in the document]
Comment (fengypants) [reply microsite/blog/2022-03-16-backstage-turns-two.md:10]: We are all over the place with this on the blog — but agree, should probably be one of these: 

`TLDR:`
`TL;DR` 
`TL;DR:`

But not what's in there.

**Pull request merged**